kserve/hack/build-kserve-manifests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# This script is used to build upstream KServe manifests
+# as kfcli uses an outdated kustomize version which cannot
+# process some features like `replacements` in the KServe manifests.
+#
+# Usage:
+# $ hack/build-kserve-manifests.sh
+
```

Comment:
```suggestion
set -euo pipefail

```

recommended for avoid otherwise undetected errors.

Reply:
well, just saw that the script is not that huge :-), so might not be worth (still, I add this to every script by habit)

Reply:
I will add `-E` for tracing too, we used it in our scripts.